.cloudbuild/library_generation/library_generation.Dockerfile

```diff
@@ -88,24 +88,23 @@ WORKDIR /protoc
 RUN source /src/library_generation/utils/utilities.sh \
 	&& download_protoc "${PROTOC_VERSION}" "${OS_ARCHITECTURE}"
 # we indicate protoc is available in the container via env vars
-ENV DOCKER_PROTOC_LOCATION=/protoc
+ENV DOCKER_PROTOC_LOCATION=/protoc/protoc/bin
```

Comment:
Is duplicated `protoc` expected in the location?

Reply:
Yes.

Do you think we should reduce the duplicated directory?

Reply:
Yes, if it is not required, just `protoc/bin` is clearer IMO. I think we can unzip to the root folder in the [line below](https://github.com/googleapis/sdk-platform-java/pull/3390/files#diff-c6e48602f6abf1dd0e120f8ee5e22e146db761c31a44d02830b67ecf17f88309R108)?

Reply:
Done.